openapi.yaml

```
@@ -17305,6 +17459,68 @@ components:
           x-linode-cli-color:
             None: black
             default_: white
+    LinodeConfigInterface:
+      type: object
+      description: >
+        The Network Interface to apply to this Linode's configuration profile.
+      properties:
+        label:
+          type: string
+          minLength: 0
+          maxLength: 64
+          pattern: '/[a-z0-9-]+/'
+          description: |
+            The name of this interface.
+
+            Only used for type `vlan` interfaces. Enter an empty string for type `public` interfaces.
+
+            Interface labels must be unique within the data center region.
```

```suggestion
            Interface labels must be unique within this Linode's configuration profile.
```
Customers can and should add the same label to different linode configs, but they must be unique within a config.